lib/crystalball/case_map.rb

```diff
@@ -8,8 +8,8 @@ class CaseMap
 
     delegate %i[push] => :affected_files
 
-    # @param [String] id of example
-    # @param [Array<String>] list of files affected by example
+    # @param [String] example - id of example
```

Comment:
its not an id its an example itself

Reply:
Then it was wrong before, and also shouldn't be a `String`

Reply:
It was wrong, but it is a string. E.g.`my_spec.rb[1:2:1]`